src/hieroglyph/builder.py

```diff
@@ -44,7 +49,7 @@ def building_slides(app):
 
 class AbstractSlideBuilder(object):
 
-    format = 'slides'
+    format = 'html'
```

Comment:
I'm pretty sure this is going to break the `ifslides` directive; your reasoning makes sense to me -- that this _is_ a type of HTML, so my claiming it here graphviz, etc will work -- but I'd like to avoid breaking other functionality to get there. Thoughts?

Reply:
At a cursory look the directive appears to check the base class of the builder:

https://github.com/nyergler/hieroglyph/blob/ea690dd7803a68ce9c4d35ded48a50da991cb15a/src/hieroglyph/builder.py#L43-L46